### PR TITLE
Adds Kitty Items walkthrough videos

### PR DESCRIPTION
**Pull request by muttoni**
## Description

Adds embeds of published walkthrough videos to the Kitty Items docs pages
______

For contributor use:

- [x] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: docs/content/kitty-items/install.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ description: Set up the project locally
 sidebar_title: Install
 ---
 
+
+## Walkthrough Video
```
Comment (alxflw) [inline docs/content/kitty-items/install.md:8]: nit: every other title is "Video Walkthrough"

File: docs/content/kitty-items/index.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ The following chapters use the project to explain the components and concepts of
 - add a new kind of NFT to the marketplace
 - interact with the Flow blockchain from within a web application
 
+## Video Walkthrough
```
Comment (alxflw) [inline docs/content/kitty-items/index.md:16]: nit: Maybe describe in one sentence what this is and that it run through the same steps laid out on each page?

**Review by alxflw — APPROVED** [review]
Left two nitpicks but approving so you can merge once reviewed

File: docs/content/kitty-items/install.md (hunk 1 above)
Comment (muttoni) [reply docs/content/kitty-items/install.md]: good catch, fixed!

File: docs/content/kitty-items/index.md (hunk 2 above)
Comment (muttoni) [reply docs/content/kitty-items/index.md:16]: Sounds good, I added an introduction outlining the format and explicitly stated that the videos are optional (depending on learning style)

**Pull request merged**